src/NUnitEngine/nunit.engine/Runners/MasterTestRunner.cs

```diff
@@ -38,8 +38,12 @@ public class MasterTestRunner : ITestRunner
         private const string TEST_RUN_ELEMENT = "test-run";
         private readonly ITestEngineRunner _engineRunner;
         private readonly IServiceLocator _services;
+#if NET20
```

Comment:
This is the only file which does `#if NET20` rather than `#if !NET_STANDARD`. We should switch and be consistent. (My bad, this one!)